proto/brpb.proto

```diff
@@ -709,6 +709,9 @@ message DataFileInfo {
     // The compression type for the file.
     CompressionType compression_type = 18;
 
+    // The CRC64XOR of all contents of the file.
+    uint64 crc64xor = 19;
```

Comment:
Dup with `bytes sha256 = 1;`?

Reply:
The `sha256` is for the whole binary file(compressed). And the `crc64xor` is composed by each individual keys. The semantic is pretty like them in the `File` message.